src/components/Api.js

```diff
@@ -0,0 +1,212 @@
+import Section from './Section.js';
+import {createNewCard} from '../utils/utils.js';
+import {cardContainerClass} from '../utils/constants.js';
+import { userInfo, formValidatorCardModal } from '../pages/index.js';
```

Comment:
Please, delete all imports

From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.